src/sentry/rules/actions/notify_event_service.py

```diff
@@ -192,6 +194,10 @@ def after(
                 self.logger.info("rule.fail.should_notify", extra=extra)
                 return
 
+            extra["organization_id"] = self.project.organization_id
+            extra["project_id"] = self.project.id
+            self.logger.info("rules.plugin_notification_sent", extra=extra)
+
             metrics.incr("notifications.sent", instance=plugin.slug, skip_internal=False)
```

Comment:
can we use the metric that is being sent to verify the notifications are being sent?

Reply:
unfortunately not, since we need to be able to distinguish these notifications from the "Send notification to all legacy integrations" action (they use the same metric)